charts/vault/templates/vault-crb.yaml

```diff
@@ -1,13 +1,14 @@
+{{- if not .Values.plugins.readKuberenetsSecrets.namespaces }}
```

Comment:
I suggest to replace by something more explicit than the no namespaces case (e.g., a different flag, a `*` namespace, etc.).